tests/serialize_test.py

```diff
@@ -44,6 +44,12 @@ def test_long_blobs(self):
             t1 = text[:_]
             self.check_serde(t1)
 
+    def test_blob_limit(self):
+        s = ' ' * 0x400000001
```

Comment:
it seems this doesn't work on CI. I imagine we will never support atoms this large anyway. Would it make sense to alter the serialization format to have a lower limit?